resotocore/resotocore/db/db_access.py

```diff
@@ -86,30 +91,72 @@ async def start(self) -> None:
         await self.pending_deferred_edge_db.create_update_schema()
         await self.package_entity_db.create_update_schema()
         for graph in cast(List[Json], self.database.graphs()):
-            log.info(f'Found graph: {graph["name"]}')
-            db = self.get_graph_db(graph["name"])
+            graph_name = GraphName(graph["name"])
+            log.info(f"Found graph: {graph_name}")
+            db = self.get_graph_db(graph_name)
             await db.create_update_schema()
+            await self.get_graph_model_db(graph_name)
         await self.cleaner.start()
 
     async def stop(self) -> None:
         await self.cleaner.stop()
 
-    async def create_graph(self, name: GraphName) -> GraphDB:
+    def graph_model_name(self, graph_name: GraphName) -> str:
+        return f"{graph_name}_model"
+
+    async def create_graph(self, name: GraphName, validate_name: bool = True) -> GraphDB:
+        if validate_name:
+            check_graph_name(name)
+
         db = self.get_graph_db(name, no_check=True)
         await db.create_update_schema()
         return db
 
+    async def copy_graph(self, source: GraphName, destination: GraphName, validate_name: bool) -> GraphName:
```

Comment:
Can we move the graph operations into GraphManager?
DbAccess is only the entry point for all db-based operations.